src/java/com/puppetlabs/http/client/impl/PersistentAsyncHttpClient.java

```diff
@@ -15,13 +15,16 @@
     private CloseableHttpAsyncClient client;
     private MetricRegistry metricRegistry;
     private String metricNamespace;
+    private Boolean useURLMetrics;
 
     public PersistentAsyncHttpClient(CloseableHttpAsyncClient client,
                                      MetricRegistry metricRegistry,
-                                     String metricNamespace) {
+                                     String metricNamespace,
+                                     Boolean useURLMetrics) {
```

Comment:
Sounds like these fails had something to do with the state of @justinstoller's machine for some reason.  After cleaning / recloning, he wasn't able to reproduce it.  Also, Travis and a few local runs on my machine didn't encounter any of these failures either, so I think we can move forward with this change.